Description: Disables the broken verifydesign tests (already disabled in the Maven POM)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -157,8 +157,8 @@
 	</target>
 		
 	<target name="clean">
-                <mkdir dir="${target.dir}" />
-                <mkdir dir="${dist.dir}" />
+                <mkdir dir="${target.dir}" />
+                <mkdir dir="${dist.dir}" />
 		<delete includeemptydirs="true">
 			<fileset dir="${target.dir}" includes="**/*" />
 			<fileset dir="${dist.dir}" includes="**/*" />
@@ -286,6 +286,7 @@
 	        <fileset dir="${target.test-classes.dir}" excludes="${skip-tests}">
 	          <exclude name="**/antclipse/**" unless="eclipse.present"/>
 	          <exclude name="walls/**"/>
+	          <exclude name="**/design/**"/>
 	        </fileset>
 	      </batchtest>
 	      <sysproperty key="antcontrib.jar" file="${target.dir}/${jar.name}"/>
